app/page.js

```diff
@@ -107,10 +111,13 @@ export default function Home() {
       secondary: {
         main: '#80CBC4',
       },
+      background: {
+        default: '#121212'
```

Comment:
Change the line with bgcolor={darkMode ? '#121212' : '#C7EBDF'} to bgcolor={'background.default'}